### PR TITLE
Edited section on Crafting Password Rules…

### PR DESCRIPTION
**Pull request by CasualDeveloper**
…to provide more clarification on the process by adding examples and linking to the Apple Developer Document on [Customizing Password AutoFill Rules](https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules).

### Overall Checklist
- [X] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)
- [X] The top-level JSON objects are sorted alphabetically
- [X] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,9 +18,11 @@ Contributing a password rule is appropriate if a service doesn't accept a passwo
 
 #### Crafting a Rule
 
-To create a password rule, you'll want to gather as much information as you can about what the website considers to be an acceptable password. You can learn this by reading any pre-stated requirements or error messages you see while experimenting with the website. Sometimes, websites don't do a great job of saying what their rules are, or their stated rules are inaccurate, and you have to determine their actual rules experimentally.
+To create a password rule, you'll want to gather as much information as you can about what the website considers to be an acceptable password. You can learn this by reading any pre-stated requirements or error messages you see while experimenting with the website. Sometimes, the rules are stated in a separate `div` element but not in the `input` element itself. Other times, websites don't do a great job of saying what their rules are, or their stated rules are inaccurate, and you have to determine their actual rules experimentally.
```
[the remark this answers is not in the document]
Comment (CasualDeveloper) [reply CONTRIBUTING.md:21]: Hi Ricky, what I meant by this was, if the site already displayed the password requirements, but only as text to the user near the input field, e.g:
html```
<div>Password Rules:
<br>Allowed: Alphanumeric only
<br>Minimum length: 8 characters
<br>Maximum length: 12 characters
</div>
<input type=password name=my-password>
```
as opposed to the site also including the rules in the input element like so:
html```
<input type=password name=my-password minlength=8 maxlength=12 pattern="^[a-zA-Z0-9]+$">
```

I think we can remove that section, if it muddies the sentence.

[the remark this answers is not in the document]
Comment (rmondello) [reply CONTRIBUTING.md:21]: We should either remove it, or clarify about it being “human-readable, not machine-readable”, or something like that. As-is, mentioning `div`, it’s confusing.

I’d be happy to take a look at another iteration of this! I love the first change!

**Comment by CasualDeveloper** [comment]
> We should either remove it, or clarify about it being “human-readable, not machine-readable”, or something like that. As-is, mentioning `div`, it’s confusing.
> 
> I’d be happy to take a look at another iteration of this! I love the first change!

That's a good suggestion! Using less technical language improves accessibility while providing clarity. Changes were made, thanks!

**Review by rmondello — APPROVED** [review]

**Comment by rmondello** [comment]
Thank you @CasualDeveloper!

**Pull request merged**